addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -127,6 +127,14 @@ export function toNormalizedPivotValue(field, groupValue, aggregateOperator) {
  * that we need in spreadsheet (display_name, isUsedInSheet, ...)
  */
 export class SpreadsheetPivotModel extends PivotModel {
+    /**
+     * Mapping `[groupBy][dateValueInReadGroup]: normalizedDateValue`.
```

Comment:
maybe add an explicit example with "March 2020" brol to be extra clear in a blink of an eye